setup.py

```diff
@@ -46,18 +49,22 @@
 extras_require['tests'] = extras_require['tests_core'] + [
     'dask',
     'ibis-framework',  # Mapped to ibis-sqlite in setup.cfg for conda
-    'spatialpandas',
     'xarray >=0.10.4',
     'networkx',
-    'datashader >=0.11.1',
     'shapely',
     'ffmpeg',
     'cftime',
     'scipy',
     'selenium',
-    'ipython >=5.4.0',
 ]
 
+# Packages not working on python 3.11 becauase of numba
+if sys.version_info < (3, 11):
+    extras_require['tests'] += [
```

Comment:
Alright yes I guess it's the best option. Let's not forget to remove that as soon as we can!

Reply:
I will open a new PR with the revert as soon as this PR is merged!

Reply:
Great thanks!